src/tests/functional/plugin/cpu/subgraph_tests/src/input_tensor_roi.cpp

```diff
@@ -0,0 +1,125 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "ngraph_functions/builders.hpp"
+#include "test_utils/cpu_test_utils.hpp"
+#include "functional_test_utils/ov_plugin_cache.hpp"
+
+using namespace ngraph;
+using namespace ngraph::op;
+using namespace InferenceEngine;
+using namespace CPUTestUtils;
+
+struct InputTensorROIParamType {
```

Comment:
Could you please enclose test classes into the `CPUSubgraphTestsDefinitions` namespace?